api/v1beta1/ccruntime_types.go

```diff
@@ -319,6 +319,7 @@ type RuntimeClass struct {
 	Name string `json:"name"`
 	// The snapshotter to be used by the runtime class
 	Snapshotter string `json:"snapshotter"`
+	PullType    string `json:"pulltype"`
```

Comment:
It'd be nice to describe the PullType property here (IIUC it should get passed to docs that way, right?)

Reply:
Also the main RuntimeClass could be updated as it states "name and snapshotter" while now there are more properties... (it should probably be just generic rather than list of properties, though)

Reply:
Fixed with a force-push.